src/Chapter2.hs

```diff
@@ -336,7 +339,11 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ _ [] = []
+subList start end xs
+  | start < 0 || end < 0 = []
+  | end < start = []
+  | otherwise = take (end-start+1) $ drop start xs
```

Comment:
Yeah, that's right! 👍🏼 

In Haskell, functions and operators are always separated from arguments with spaces. And binary operators with spaces on both sides ◀️ 👾 ▶️ 

```suggestion
  | otherwise = take (end - start + 1) $ drop start xs
```

Reply:
Thanks for the suggestion